app/2.1.x/db-less-admin-api.md

```diff
@@ -511,9 +527,9 @@ upstream_data: |
 target_body: |
     Attributes | Description
     ---:| ---
-    `target` |  The target address (ip or hostname) and port. If the hostname resolves to an SRV record, the `port` value will be overridden by the value from the DNS record.
+    `target` |  The target address (ip or hostname) and port. If the hostname resolves to an SRV record, the `port` value will be overridden by the value from the DNS record. 
     `weight`<br>*optional* |  The weight this target gets within the upstream loadbalancer (`0`-`1000`). If the hostname resolves to an SRV record, the `weight` value will be overridden by the value from the DNS record.  Defaults to `100`.
-    `tags`<br>*optional* |  An optional set of strings associated with the Target, for grouping and filtering.
+    `tags`<br>*optional* |  An optional set of strings associated with the Target, for grouping and filtering. 
```

Comment:
global comment-don't need a comma here because it creates an incomplete clause.

```suggestion
    `tags`<br>*optional* |  An optional set of strings associated with the Target for grouping and filtering. 
```
Or reorder all of these:

An optional set of strings **for grouping and filtering** that are associated with the Target.